src/components/song/Song.jsx

```diff
@@ -1,15 +1,24 @@
-import React from "react";
+import React from 'react'
+import PropTypes from 'prop-types'
+import './_Song.scss'
+import AlbumArt from '../common/album-art/AlbumArt'
```

Comment:
Use the alias instead. Avoid using relative paths 😀

Reply:
Attended